mkdocs.yml

```diff
@@ -132,6 +132,7 @@ nav:
       - software-platforms/lsfr/index.md
     - Amazon S3:
```

Comment:
In my opinion, this needs to be rearranged a bit. When it got stubbed in, it was assumed that the header would be S3, but LfR is also AWS! The AWS index page talks about signing up, which is pretty important!

Reply:
@dvenprasad I think it's probably ok to just remove the `lsfr` directory altogether, and just put all those pages into the `aws` directory?

Reply:
The 25th iteration of our IA has landed 😂 #61

Reply:
Okay maybe we take a pass at rearranging all at once? The relative link headache will be real.